src/listener/Heading.php

```diff
@@ -27,10 +27,19 @@ public function process(Line $line)
 
     /**
      * {@inheritDoc}
+     * 
+     * @since 1.2.0 Added exception
+     * 
+     * @throws \Exception for unknown heading levels
```

Comment:
Please add Exception into use statement therefore this backslash won't be need. 

No newline ;-)